notebooks/Notebook3.md

```diff
@@ -18,17 +20,72 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+I made some pretty big design changes.
+
+
+*Chapter One:*
+
+Fluency, and allowing method chaining. For operators like "-", I ensured that they took in AND returned the unorderedString type. By doing so, you can chain methods and you don't have to worry about it popping out of unorderedString. 
+
+
+*Chapter Two:*
+
+I also added in "extension(s :String)" to, essentially, allow easy conversion from String to either of the new types. To denote which type you want, explicitly. (Without having to do the whole "unorderedString("hello world");")
+
+var v1 = "hello world" u   <-- v1 will be an unorderedString. 
+
+var v2 = "hello world"/ <---- v2 will be a PhonemicString.
+
+So, you still have all the implicit conversions to get in and out of types, and so - if you use any of the UnorderedString operators on string, it will automatically switch into the correct type and perform the operation. But, if you use an operator that is in both String and unorderedString, like "+", you can chose which route you want to go by using "u". I have also added an implicit conversion to String, so you can use String operators on UnorderedString and it will pop into String, do the operation, and pop back into unorderedString.
```

Comment:
This is a smart way of handling conversions and seems a lot simpler and easy to understand for the users. That was also smart to add an implicit conversion. Did you also do this for phenomic strings or just unordered?